setup.py

```diff
@@ -27,6 +27,7 @@
         "mmh3>=2.0,<=3.0.0",
         "pandas==1.1.5; python_version<'3.7'",
         "pandas>=1.2.4,<=1.3.5; python_version>'3.6'",
+        "pandas>=1.2.4,<=1.4.1; python_version>='3.8'",
```

Comment:
I've tested with "pandas>=1.2.4; python_version>='3.6'", and it worked perfectly fine.

So please remove L30 and alter L29 into
```
"pandas>1.2.4; python_version>3.6"
```

Reply:
thanks @XuanYang-cn. Just make sure when building the package it is built with different python version, as the new pandas > 1.4 only works with python 3.8 and above

Reply:
Yes, pip will figure it out automatically. With python3.7, pandas1.4.1 cannot be downloaded, you could try locally.

pip will only download the highest pandas version that satisfies python3.7, which is 1.3.5. 

Here's a ref https://pypi.org/project/pandas/1.4.1/ , please pay attention the programming language part.

![image](https://user-images.githubusercontent.com/51370125/158747534-29d6187b-c319-4032-a484-ada03c707800.png)